### ISSUE TITLE
Project License?

### ISSUE THREAD

**Issue by kphillisjr** [jharler/zt_sln#1]
I was looking over the code in the project and was wondering what the License for this project is. Is the license the same as the main ZeroTolerance Project?

**Comment by jharler** [comment]
Yes, the license is the same as the ZeroTolerance project.  I'll update this project with a license.  Thanks for pointing out that it's missing.
